etc/config/numba.defaults.properties

```diff
@@ -0,0 +1,17 @@
+compilers=&numba
+defaultCompiler=numba61
+
+group.numba.compilers=numba61
+group.numba.isSemVer=true
+group.numba.baseName=Numba
+# TODO(Rupt) build environments
+compiler.numba61.exe=/TODO/bin/python3.12
```

Comment:
Incomplete

Reply:
>how can we help get this over the finish line?

@mattgodbolt this is the part I want help with. How should we arrange the python virtual environment?

Reply:
Got it!

We can probably now install using our newer installation system that supports python venvs. For example if you look at something like [vyper](https://github.com/compiler-explorer/infra/blob/main/bin/yaml/vyper.yaml) you'll see it uses `pip` as its install method, referenes the version of python it needs and installs various versions. Do you think that would work?

If `pip` doesn't work we can consider trying to get `conda` working though that's a bit more tricky as the machine we run installation on is not the same as the one we run things on, and in my experience conda can be fussy about moving things after. I reckon we can get it working though.

Reply:
Thank you! Conda should not be needed.

Reply:
- https://github.com/compiler-explorer/infra/pull/1530